src/auth/auth.controller.spec.ts

```diff
@@ -7,6 +7,8 @@ import { UsersService } from '../users/users.service';
 import { AuthController } from './auth.controller';
 import { AuthService } from './auth.service';
 import { LoginReqDto } from './models/dto/auth.dto';
+import { AuthenticationGuard } from './guards/auth.guard';
+import { boolean } from '@hapi/joi';
```

Comment:
Codacy found an issue: ['boolean' is defined but never used.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-backend/pullRequest?prid=5747194)